.github/pull_request_template.md

```diff
@@ -1,26 +1,25 @@
-<!--
-Add link to the connected issue.
-If this PR closes the issue, use:
+### Proposed changes
 
-Fixes #<NUMBER>
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+If it fixes a bug or resolves a feature request, be sure to link to that issue._
 
-If not, use (or similar):
+#### Type of change
 
-Related to #<NUMBER>
--->
+- [ ] 🐛 Bugfix (change which fixes an issue)
+- [ ] 🚀 Feature (change which adds functionality)
+- [ ] 📚 Documentation (change which fixes or extends documentation)
 
-## Summary
-<!--
-For bugs, describe how this PR solves the issue, how to check if it's fixed, etc.
-For features request, describe shortly what's this solving.
+💥 **Breaking change!** _Explain why a non-backwards compatible change is necessary or remove this line entirely if not applicable._
```

Comment:
Feel free to ignore:
```suggestion
💥 **Is this a change that can break compatibility for other users?** _If the answer is yes, please elaborate about why a non-backwards compatible change is necessary._
```

Reply:
I'll ignore this one, I like the current version :man_shrugging:.